### PR TITLE
[DOC] Add traces to profile content

### LINKED ISSUE #2662: [DOC] Add traces to profile content to Pyroscope OSS
Create a new page in the Pyroscope does that covers profiling + other signals (like traces). 

We may want to include basic information like the Traces to profiles section from grafana/explore/trace-integration.md.

Reference: https://github.com/grafana/grafana/pull/77844/files#r1388353845 

### Traces to profiles in Grafana

We have content for Traces to profiles content available as a shared file in https://github.com/grafana/grafana/blob/main/docs/sources/shared/datasources/tempo-traces-to-profiles.md 

If this file applies to the Pyroscope, we can use the shared file in the Pyroscope OSS doc without having to rewrite that content. 

This content is currently used here: https://grafana.com/docs/grafana-cloud/connect-externally-hosted/data-sources/tempo/configure-tempo-data-source/#trace-to-profiles

### PR DESCRIPTION
**Pull request by knylander-grafana**
Adds traces to profiles content from Grafana OSS and restructures the invest and analyze profile pages. 
* Ingest and analyze folder is renamed to View and analyze
* Flamegraphs are also moved underneath the View and analyze section. 
* Profiling and traces are moved to the View and analyze section
* Added a new page for Traces to profile content (shared file from Grafana)

This is the updated table of contents with the traces to profiles content: 

image

Fixes #2662

### CODE REVIEW

File: docs/sources/view-and-analyze-profile-data/_index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Options for Visualizing Data in Grafana:
+- **Pyroscope App Plugin**: This plugin is specifically designed for Pyroscope data. It allows for easy browsing, analysis, and comparison of multiple profiles across different labels or time periods. This is particularly useful for a comprehensive overview of your application's performance.
+- **Explore Tab**: In Grafana, the Explore tab is suited for making targeted queries on your profiling data. This is useful for in-depth analysis of specific aspects of your application's performance.
+- **Dashboard**: Grafana dashboards are excellent for integrating profiling data with other metrics. You can display Pyroscope data alongside other dashboard items, creating a unified view of your application’s overall health and performance.
+
+For more information on using profiles in Grafana, refer to [Pyroscope and profiles in Grafana](https://grafana.com/docs/pyroscope/latest/introduction/pyroscope-in-grafana/#pyroscope-and-profiling-in-grafana).
```
Comment (knylander-grafana) [inline docs/sources/view-and-analyze-profile-data/_index.md:38]:
```suggestion
For more information on using profiles in Grafana, refer to [Pyroscope and profiles in Grafana]({{< relref "../introduction/pyroscope-in-grafana/#pyroscope-and-profiling-in-grafana" >}}).
```

File: docs/sources/view-and-analyze-profile-data/_index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- **Flamegraphs**: Visualize call relationships and identify hotspots.
+- **Tables**: View detailed statistics for specific functions or time periods.
+- **Charts and graphs**: Analyze trends and compare performance across different metrics.
+
+## Viewing profiles
+
+Pyroscope offers both a Command Line Interface (CLI) and an Application Programming Interface (API) to interact with and retrieve profiling data. These tools provide flexibility in how you access and manage your profiling information.
+
+You can export profiling data from Pyroscope in various formats:
+- **JSON**: JSON object easy to integrate with other tools and scripts
```
Comment (cyriltovena) [inline docs/sources/view-and-analyze-profile-data/_index.md:28]: The most popular is pprof and we support that to be precised it gzip compressed pprof `foo.pprof.gz`

**Review by cyriltovena — APPROVED** [review]
LGTM

File: docs/sources/configure-server/about-server-api.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
 ---
 description: Learn about the Pyrocope server API
-menuTitle: Server API Overview
+menuTitle: Server API overview
 title: Pyroscope Server HTTP API Reference
 weight: 20
 ---
 
-# Pyroscope Server HTTP API Reference
+# Pyroscope server HTTP API reference
```
Comment (knylander-grafana) [inline docs/sources/configure-server/about-server-api.md:8]:
```suggestion
# Pyroscope server HTTP API
```

File: docs/sources/configure-server/about-server-api.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
 ---
 description: Learn about the Pyrocope server API
-menuTitle: Server API Overview
+menuTitle: Server API overview
 title: Pyroscope Server HTTP API Reference
```
Comment (knylander-grafana) [inline docs/sources/configure-server/about-server-api.md:4]:
```suggestion
title: Pyroscope Server HTTP API
```

File: docs/sources/view-and-analyze-profile-data/_index.md (hunk 2 above)
Comment (knylander-grafana) [inline docs/sources/view-and-analyze-profile-data/_index.md:28]:
```suggestion
- **pprof**: Support for pprof, gzip compressed pprof, `foo.pprof.gz`
- **JSON**: JSON object easy to integrate with other tools and scripts
```

File: docs/sources/view-and-analyze-profile-data/profile-tracing/traces-to-profiles.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+menuTitle: Traces to profiles
+description: Learning about traces to profiles integration in Grafana and Grafana Cloud.
+weight: 150
+keywords:
+  - pyroscope
+  - continuous profiling
+  - tracing
+---
+
+# Trace to profiles
```
Comment (knylander-grafana) [inline docs/sources/view-and-analyze-profile-data/profile-tracing/traces-to-profiles.md:12]:
```suggestion
# Traces to profiles
```

File: docs/sources/configure-server/about-server-api.md [hunk 6]
```diff
@@ -1,17 +1,17 @@
 ---
 description: Learn about the Pyrocope server API
-menuTitle: Server API Overview
-title: Pyroscope Server HTTP API Reference
+menuTitle: Server HTTP API 
+title: Pyroscope Server HTTP API
```
Comment (knylander-grafana) [inline docs/sources/configure-server/about-server-api.md:4]:
```suggestion
title: Pyroscope server HTTP API
```

File: docs/sources/view-and-analyze-profile-data/_index.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+Options for Visualizing Data in Grafana:
+- **Pyroscope App Plugin**: This plugin is specifically designed for Pyroscope data. It allows for easy browsing, analysis, and comparison of multiple profiles across different labels or time periods. This is particularly useful for a comprehensive overview of your application's performance.
+- **Explore Tab**: In Grafana, the Explore tab is suited for making targeted queries on your profiling data. This is useful for in-depth analysis of specific aspects of your application's performance.
+- **Dashboard**: Grafana dashboards are excellent for integrating profiling data with other metrics. You can display Pyroscope data alongside other dashboard items, creating a unified view of your application’s overall health and performance.
+
+For more information on using profiles in Grafana, refer to [Pyroscope and profiles in Grafana]({{< relref "../introduction/pyroscope-in-grafana/#pyroscope-and-profiling-in-grafana" >}}).
```
Comment (knylander-grafana) [inline docs/sources/view-and-analyze-profile-data/_index.md:38]:
```suggestion
For more information on using profiles in Grafana, refer to [Pyroscope and profiles in Grafana]({{< relref "../introduction/pyroscope-in-grafana#pyroscope-and-profiling-in-grafana" >}}).
```

File: docs/sources/view-and-analyze-profile-data/_index.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+Options for Visualizing Data in Grafana:
+- **Pyroscope App Plugin**: This plugin is specifically designed for Pyroscope data. It allows for easy browsing, analysis, and comparison of multiple profiles across different labels or time periods. This is particularly useful for a comprehensive overview of your application's performance.
+- **Explore Tab**: In Grafana, the Explore tab is suited for making targeted queries on your profiling data. This is useful for in-depth analysis of specific aspects of your application's performance.
+- **Dashboard**: Grafana dashboards are excellent for integrating profiling data with other metrics. You can display Pyroscope data alongside other dashboard items, creating a unified view of your application’s overall health and performance.
+
+For more information on using profiles in Grafana, refer to [Pyroscope and profiles in Grafana]({{< relref "../introduction/pyroscope-in-grafana#pyroscope-and-profiling-in-grafana" >}}).
+
+Note the Pyroscope app plugin is cloud only...
```
Comment (knylander-grafana) [inline docs/sources/view-and-analyze-profile-data/_index.md:40]:
```suggestion
The Pyroscope app plugin works for Grafana Cloud. 
```

File: docs/sources/view-and-analyze-profile-data/_index.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+- **Tables**: View detailed statistics for specific functions or time periods.
+- **Charts and graphs**: Analyze trends and compare performance across different metrics.
+
+## Viewing profiles
+
+Pyroscope offers both a Command Line Interface (CLI) and an Application Programming Interface (API) to interact with and retrieve profiling data. These tools provide flexibility in how you access and manage your profiling information.
+
+You can export profiling data from Pyroscope in various formats:
+- **JSON**: JSON object easy to integrate with other tools and scripts
+- **dot**: text containing a DOT representation of the profile
```
Comment (kolesnikovae) [inline docs/sources/view-and-analyze-profile-data/_index.md:29]: We used to have this option in pre v1 versions, but as of now there is no way to visualize/export profiles in the dot format in Grafana

Comment (Rperry2174) [reply docs/sources/view-and-analyze-profile-data/_index.md:29]: really? I thought we added it for the hackathon cc @aleks-p

Comment (aleks-p) [reply docs/sources/view-and-analyze-profile-data/_index.md:29]: There is support in the `/render` endpoint if `format=dot` is passed. Anton is right that we can't visualize it (e.g. with Graphviz) or export it to a file yet.

Comment (Rperry2174) [reply docs/sources/view-and-analyze-profile-data/_index.md:29]: makes sense @knylander-grafana we should remove `dot` from here our options we should list here are `json`, `pprof` 
image

(png is actually broken right now)

Comment (knylander-grafana) [reply docs/sources/view-and-analyze-profile-data/_index.md:29]: Okay. I'll update it. Thank you! Glad we are doing this review!

Comment (knylander-grafana) [inline docs/sources/view-and-analyze-profile-data/_index.md:29]:
```suggestion
```

File: docs/sources/view-and-analyze-profile-data/_index.md (hunk 2 above)
Comment (knylander-grafana) [reply docs/sources/view-and-analyze-profile-data/_index.md:29]: Added.

**Comment by knylander-grafana** [comment]
Note that there is a link to a page that is created in #2982. The doc validator won't pass because of this broken link (until the other PR is merged).

**Pull request merged**